backend/src/documents/extract_text.py

```diff
@@ -0,0 +1,58 @@
+import json
```

Comment:
Do you think the refactor work I did for `src/external/lambdas/helpers/s3_file_upload_helper.py` moving someplace like `src/storage/cloud_upload.py` or something of that nature?

Reply:
I wasn't thinking `storage`, but in `documents`.  So, `src/documents/new_document.py`, `upload_document.py`, or something like that.